index.html

```diff
@@ -86,5 +86,197 @@ <h1>I'm Stephen Glad to see you</h1>
+        <div class="abt-me flex column">
+          <h1 class="header-text">About Myself</h1>
+          <p class="para-text">
+            Hello I’m a software developer! I can help you build a product , feature or
+            website Look through some of my work and experience! If you like what you see
+            and have a project you need coded, don’t hestiate to contact me.
+          </p>
+          <div class="self-container">
+            <p>Let's Connect</p>
+            <div class="list-nav">
+              <ul class="list-items skill-items">
+                <li>
+                  <a href="">
+                    <img
+                      src="./images/twitter.png"
+                      alt="my social handle images"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="">
+                    <img
+                      src="./images/Linkedin-icon.svg"
+                      alt="my social handle images"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="">
+                    <img
+                      src="./images/medium.png"
+                      alt="my social handle images"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="">
+                    <img
+                      src="./images/github.png"
+                      alt="my social handle images"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href=""><img src="./images/angellist.png"
+                      alt="my social handle images"
+                    />
+                  </a>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <button type="button">Get my Resume</button>
+        </div>
+        <div class="language flex column">
+          <div class="lang flex column">
+            <p class="lang-text flex"><span class="txt">Language</span><span class="lang-line">></span></p>
+            <ul class="lang-div flex column">
+              <li class="lang-list flex">
+                <img src="./images/javaScript.png" alt="js logo">
+                <p class="skill-text">JavaScript</p>
+              </li>
+              <li class="lang-list flex">
+                <img src="./images/html.png" alt="js logo">
+                <p class="skill-text">HTML</p>
+              </li>
+              <li class="lang-list flex">
+                <img src="./images/css.png" alt="js logo">
+                <p class="skill-text">CSS</p>
+              </li>
+            </ul>
+          </div>
+          <div class="lang">
+            <p class="lang-text flex"><span class="txt">Framework</span><span class="line">></span></p>
+          </div>
+          <div class="frame-line">
+            <hr>
+          </div>
```

Comment:
- [ ] Kindly make this a `ul` `li` structure for semantic reasons, the final structure should look like this:
```
<ul>
  <li>Languages
    <ul>
       <li>JavaScript</li>
       ...
    </ul>
  </li>
  <li>Skills</li>
  .....
</ul>
```